tests/integration-tests/conftest.py

```diff
@@ -1407,14 +1407,14 @@ def odcr_stack(
         AvailabilityZone=availability_zone_1,
         InstanceCount=2,
         InstancePlatform="Linux/UNIX",
-        InstanceType="t3.micro",
+        InstanceType="c5.xlarge",
```

Comment:
Why are we changing the instance types?

Reply:
Because .micro instance types is too small for RHEL9. RHEL 9 requires 1.5GB of memory. t3.micro has 1 GB

See https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/9/html/performing_a_standard_rhel_9_installation/system-requirements-reference_installing-rhel